### PR TITLE
Bc1 66 review all parts of the project and fix the bugs

### PR DESCRIPTION
**Pull request by MaxRichard2007**

### CODE REVIEW

File: app-land/scss/layout/_planSelectorHub.scss [hunk 1]
```diff
@@ -184,151 +184,132 @@
...
-    */
-
   @include tablet {
     .plans {
-      .download-box {
-        .information-section {
-          width: 100%;
-          padding: 0;
+      padding: 2rem;
+      border-radius: 10px;
```
Comment (sadiGFG) [inline app-land/scss/layout/_planSelectorHub.scss:197]: Please use rem or em

File: app-land/scss/layout/_planSelectorHub.scss [hunk 2]
```diff
@@ -184,151 +184,132 @@
...
+
+        .image-box {
+          display: none;
+        }
+
         .information-section {
-          .information-box {
-          }
+          width: 100%;
+          padding: 2rem 1rem;
```
Comment (sadiGFG) [inline app-land/scss/layout/_planSelectorHub.scss:261]: use our shared spacing.

File: app-land/scss/layout/_planSelectorHub.scss [hunk 3]
```diff
@@ -184,151 +184,132 @@
...
+              height: 2.5rem;
+              font-size: $fontSizeBase;
+            }
           }
         }
+
         .box {
-          h3,
+          flex-direction: column;
+          gap: 1rem;
```
Comment (sadiGFG) [inline app-land/scss/layout/_planSelectorHub.scss:240]: use our shared spacing

File: app-land/scss/layout/_planSelectorHub.scss [hunk 4]
```diff
@@ -184,151 +184,132 @@
...
-    */
-
   @include tablet {
     .plans {
-      .download-box {
-        .information-section {
-          width: 100%;
-          padding: 0;
+      padding: nth($spaces, 8);
+      border-radius: 10px;
```
Comment (ahmadmohammadirad2006) [inline app-land/scss/layout/_planSelectorHub.scss:197]: Please use variables from shared folder

**Review by ahmadmohammadirad2006 — APPROVED** [review]

**Review by sadiGFG — APPROVED** [review]

**Pull request merged**